tools/RAiDER/statsPlot.py

```diff
@@ -351,6 +369,11 @@ def resid(x, d, v, m):
                                    loss='soft_l1', f_scale=0.1,
                                    args=(d, v, model))
 
+        # convert range to specified output unit
+        # TODO temp fix first multiply by 1,010,000 m as crude conversion from degrees to km
+        res_robust.x[0] = res_robust.x[0]*1010000
```

Comment:
Note that equator 1 degree is roughly 110km so should be * 110,000 to go from degree to m and not 1,010,000.

Reply:
Fixed